src/components/SummaryPanel/RpsChart.tsx

```diff
@@ -51,7 +51,7 @@ export class RpsChart extends React.Component<RpsChartTypeProp, {}> {
       maxRps = sample > maxRps ? sample : maxRps;
       sumRps += +sample;
     }
-    const avgRps = len === 0 ? 0 : sumRps / (len - 1);
+    const avgRps = len < 2 ? 0 : sumRps / (len - 1);
```

Comment:
Minor, but I think this fix should be on a different commit so it can be referenced/queried if necessary.

Reply:
Guilty as charged, I have a [bad] habit of including tweaks.  I will try to be better.